code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2,6 +2,11 @@
 #define LIQUID 2
 #define GAS 3
 
+#define NORMAL 0
+#define FAST 1
```

Comment:
if you wanna use those defines elsewhere you better move them in some define dm file